apps/landing/landing.html

```diff
@@ -50,6 +50,12 @@
             <li class="nav-item  link" style="font-family: sans-serif;">
                 <a class="nav-link" href="../table.html"> <i class="fas fa-list-ul"></i>  Slides</a>
             </li>
+            <li class="nav-item link">
```

Comment:
We're now having the `info` tab in NavBar in landing.html, table.html, info.html which was removed from navbar in #376